### PR TITLE
kie-issues#225: Support context Data Type on inputs/outputs of DMN Runner

### LINKED ISSUE #225: Support `context` Data Type on inputs/outputs of DMN Runner
Now, the `context` Data Type is not supported by the DMN Runner table. Adding an input with the `context` Data Type will render a text input field, which will essentially be a `string`.  This causes to send a `string` to the JIT Executor, and the `context` value is not correctly evaluated.

### Steps do Reproduce
 - Create a new DMN file;
 - Add a Input node called `myInput` and a Decision node called `myDecision`;
 - Change the Input node type to `context`;
 - Add the following Literal Expression in the Decision node: `myInput.x`;
 - Open the DMN Runner, and fill the `myInput` field with: `{ "x": 1 }`;
 - The output value should be `1` but it's `null`;

### Example

https://user-images.githubusercontent.com/24302289/234119678-f6aa7921-2660-4dbe-8305-60abb5ae1a57.mp4

```[tasklist]
### Acceptance criteria
- [ ] The DMN Runner table should correctly evaluate the "context" type.
```

### PR DESCRIPTION
**Pull request by ljmotta**
## kie-issue:
Closes [kie-issue#225](https://github.com/kiegroup/kie-issues/issues/225)

## On this PR

 - Created a new package `dmn-runner` to share components and types between the DMN Runner form and table;
   - This enabled to remove a form dependency from the table;
 - Correctly evaulate the `context` datatype on the DMN Runner table;'
 - Improved general DMN Runner outputs table;

### DMN Runner `context` datatype
When the user creates a Input and Decision node with the `context` data type the DMN Runner outputs table will render columns using the user inputs as follow:

https://user-images.githubusercontent.com/24302289/236061448-90331ec8-e188-467a-8911-af0564fdf7ed.mp4

Some observations:
 - The plain inputs, that aren't an object are placed on the `context` column;
 - All values inside objects are infered as `Any`;

### CODE REVIEW

File: packages/unitables-dmn/src/DmnRunnerOutputs.tsx [hunk 1]
```diff
@@ -18,184 +18,47 @@ import * as React from "react";
...
 
 interface OutputField {
+  type: string;
   dataType: DmnBuiltInDataType;
   width?: number;
   name: string;
   joinedName: string;
+  properties?: Record<string, any>;
+  items?: Record<string, any>[];
 }
```
Comment (thiagoelg) [inline packages/unitables-dmn/src/DmnRunnerOutputs.tsx]:
```suggestion
interface OutputFields {
  type: string;
  dataType: DmnBuiltInDataType;
  width?: number;
  name: string;
  joinedName: string;
  properties?: Record<string, any>;
  items?: Record<string, any>[];
}
```

Then delete the `export type OutputFields = OutputField;`

**Review by thiagoelg — APPROVED** [review]
Just one minor suggestion, but other than that this looks good!

File: packages/unitables-dmn/src/DmnRunnerOutputs.tsx (hunk 1 above)
Comment (ljmotta) [reply packages/unitables-dmn/src/DmnRunnerOutputs.tsx]: @thiagoelg Thanks for your review! Actually, I forgot to remove the `OutputFields`. I've kept the `OutputField`, and removed all `OutputFields` references.

**Review by yesamer — APPROVED** [review]
LGTM @ljmotta Thank you.

**Comment by yesamer** [comment]
@ljmotta Can I merge it?

**Pull request merged**